### PR TITLE
Updates to TS SDK

### PR DESCRIPTION
**Pull request by eugene-chen**
Updated missing docs, changed references from base units to raw base units, removed axios dependency

### CODE REVIEW

File: typescript/phoenix-sdk/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ellipsis-labs/phoenix-sdk",
-  "version": "1.2.2",
+  "version": "1.2.4",
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/package.json:3]: i deployed on 1.2.3 so skipping to 1.2.4

File: typescript/phoenix-sdk/src/client.ts [hunk 2]
```diff
@@ -486,11 +489,12 @@ export class Client {
   }
 
   /**
-   * Get the price in ticks for a given price in quote units per base unit
-   * Example: For a tick size of 0.01 (quote units per base unit), a price in quote units per
-   * base unit of 1.23 would be 123 ticks
+   * Given a price in quote units per raw base unit, returns the price in ticks.
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/src/client.ts:492]: float price is in quote units per raw base unit, not quote units per base unit

**Review by jarry-xiao — APPROVED** [review]

File: typescript/phoenix-sdk/src/client.ts [hunk 3]
```diff
@@ -613,11 +627,12 @@ export class Client {
...
   /**
-   * Get the amount of base units for a given amount of base atoms
+   * Given a number of base atoms, returns the equivalent number of raw base units.
+   *
    * @param baseAtoms The amount of base atoms to convert
-   * @param marketAddress The `PublicKey` of the market account
+   * @param marketAddress The `PublicKey` of the market account, as a string
    */
-  public baseAtomsToBaseUnits(
+  public baseAtomsToRawBaseUnits(
```
Comment (jarry-xiao) [inline typescript/phoenix-sdk/src/client.ts:635]: Make sure this is changed everywhere

File: typescript/phoenix-sdk/examples/deserializeClock.ts [hunk 4]
```diff
@@ -3,10 +3,10 @@ import { BN } from "bn.js";
 
 import * as Phoenix from "../src";
 
-// Ex: ts-node tests/market.ts
-export async function market() {
-  const connection = new Connection("https://api.mainnet-beta.solana.com");
-  const phoenix = await Phoenix.Client.create(connection, "mainnet");
+// Ex: ts-node examples/deserializeClock.ts
+export async function deserializeClock() {
+  const connection = new Connection("https://api.devnet.solana.com");
```
Comment (jarry-xiao) [inline typescript/phoenix-sdk/examples/deserializeClock.ts:8]: What's wrong with getting the mainnet clock? FWIW I think the public mainnet RPC is slightly more reliable than the public devnet RPC

File: typescript/phoenix-sdk/src/client.ts [hunk 5]
```diff
@@ -486,39 +489,51 @@ export class Client {
...
+   * @param marketAddress The `PublicKey` of the market account, as a string
    */
   public floatPriceToTicks(price: number, marketAddress: string): number {
     const market = this.markets.get(marketAddress);
     if (!market) throw new Error("Market not found: " + marketAddress);
+
     return Math.round(
-      (price * 10 ** market.quoteToken.data.decimals) /
-        market.data.quoteLotsPerBaseUnitPerTick
+      (price *
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/src/client.ts:504]: Check this math @jarry-xiao

File: typescript/phoenix-sdk/src/client.ts [hunk 6]
```diff
@@ -486,39 +489,51 @@ export class Client {
...
+   * @param marketAddress The `PublicKey` of the market account, as a string
    */
   public ticksToFloatPrice(ticks: number, marketAddress: string): number {
     const market = this.markets.get(marketAddress);
     if (!market) throw new Error("Market not found: " + marketAddress);
     return (
-      (ticks * market.data.quoteLotsPerBaseUnitPerTick) /
-      10 ** market.quoteToken.data.decimals
+      (ticks *
+        market.data.quoteLotsPerBaseUnitPerTick *
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/src/client.ts:525]: Check this math too

File: typescript/phoenix-sdk/examples/deserializeClock.ts (hunk 4 above)
Comment (eugene-chen) [reply typescript/phoenix-sdk/examples/deserializeClock.ts:8]: was testing client creation w/ devnet, reverted.

File: typescript/phoenix-sdk/src/client.ts (hunk 5 above)
Comment (jarry-xiao) [reply typescript/phoenix-sdk/src/client.ts:504]: This looks fine to me, we can consider using `market.data.header.tickSizeInQuoteAtomsPerBaseUnit` in the denominator.

File: typescript/phoenix-sdk/src/client.ts (hunk 6 above)
Comment (jarry-xiao) [reply typescript/phoenix-sdk/src/client.ts:525]: This looks correct to me as well

**Pull request merged**